packages/@uppy/dashboard/src/index.js

```diff
@@ -502,6 +504,12 @@ module.exports = class Dashboard extends Plugin {
     }
   }
 
+  disableAllFocusableElements = (disable) => {
+    const focusableNodes = toArray(this.el.querySelectorAll(FOCUSABLE_ELEMENTS))
+    focusableNodes.forEach(node => node.setAttribute('tabindex', disable ? '-1' : '0'))
```

Comment:
@goto-bus-stop what do you think of this? I’ve checked and we mostly have `tabindex=0` on FOCUSABLE_ELEMENTS, nothing else is used, so not worth storing the previous value before changing it to `-1`?

P.S. TIL “[The maximum value for tabindex is 32767](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)”.